Desktop: add note about rotated key, chrome-sandbox instructions (#7231)
mattermost/docs: source/collaborate/install-desktop-app.rst

  Both a ``.deb`` package (Beta), and an official APT repository is available for Debian 9 and for Ubuntu releases 20.04 LTS or later. Automatic app updates are supported and enabled. When a new version of the desktop app is released, your app updates automatically.

  .. important::

>     The GPG public key has changed. If you had previously setup the repository on your system, you'll need to download the new key: you can set the ``UPDATE_GPG_KEY=yes`` environment variable when running the setup script, to configure it to overwrite the previous key on your system with the new one. E.g. the first step of the instructions would become ``curl -fsS -o- https://deb.packages.mattermost.com/setup-repo.sh | sudo UPDATE_GPG_KEY=yes bash``. Note that depending on your setup, additional steps may also be required, particularly for installations that didn't rely on the repository setup script.

Reviewer: ```suggestion
    The GPG public key has changed. If you had previously set up the repository on your system, you'll need to download the new key. You can set the ``UPDATE_GPG_KEY=yes`` environment variable when running the setup script to configure it to overwrite the previous key on your system with the new one. The first step of installation then becomes: ``curl -fsS -o- https://deb.packages.mattermost.com/setup-repo.sh | sudo UPDATE_GPG_KEY=yes bash``. Depending on your setup, additional steps may also be required, particularly for installations that don't rely on the repository setup script.
```

Author: Thanks for the improvements Carrie :tada:
